src/ucp/tag/tag_send.c

```diff
@@ -148,15 +148,15 @@ ucp_tag_send_inline(ucp_ep_h ep, const void *buffer, size_t count,
 
     length = ucp_contig_dt_length(datatype, count);
 
-    if ((ssize_t)length <= ucp_ep_config(ep)->tag.eager.max_short) {
+    if ((ssize_t)length <= ucp_ep_config(ep)->tag.offload.max_eager_short) {
```

Comment:
why need to change to order of if?

Reply:
to avoid sending using uct_ep_am_short when tag lane is present. 
We have to distinguish TAG and AM cases somehow, because we removed the check for offload flsg.

Reply:
can we have separate config for tag/offload and tag/am? at least max_short which is used in fast path

Reply:
i think it's better to add, since extra u32 param in config struct is probably negligible, however changing order of condition is extra fast path branch for SW TM